pavelib/paver_tests/test_eslint.py

```diff
@@ -8,16 +8,16 @@
 from paver.easy import BuildFailure
 
 
-class TestPaverJsHint(unittest.TestCase):
```

Comment:
Did you want to keep the JSHint tests since you are now keeping JSHint around? Since we know we're planning on removing it soon then maybe removing it is okay, but it might be better to keep it just in case we don't get around to doing the follow-on work.